main.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'app'
+
+def print_interface
+  puts "Please choose an option by enterin a number:
+    1 - List all books
+    2 - List all people
+    3 - Create a person
```

Comment:
- [ ] Kindly stick with the guidelines [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) in the `Create a person` case. I don't have to take the `Classroom` from the user.
![image](https://user-images.githubusercontent.com/24703171/209451957-29338703-9f9d-4517-9339-eb886ddde923.png)